spec/model/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.new(name: 'Something',
+                  photo: 'http://rrrrrrrr/something.jpg',
+                  bio: 'Something test',
+                  posts_counter: 0)
+  post = Post.new(
+    title: 'Something',
+    text: 'Something test',
+    author: user,
+    comments_counter: 0,
+    likes_counter: 0
+  )
+  subject do
+    described_class.new(
+      author: user,
+      post: post
+    )
+  end
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+end
```

Comment:
As part of the project requirements, you are suppose to add unit specs for all of your models' methods and validations. However, all the tests I see here is for the models' validations. Kindly consider adding tests for the like model's method `update_post_like_counter`

Reply:
Thank you for your review @jerryowusu 

Can you kindly provide me with more information on how to implement that?